packages/orchestration/README.md

```diff
@@ -17,9 +17,92 @@ $ npm install @sap-ai-sdk/orchestration
   - Create a `.env` file in the sample-code directory.
   - Add an entry `AICORE_SERVICE_KEY='<content-of-service-key>'`.
 
-## Usage
+## Orchestration client
 
-<!-- Details to be added -->
+Orchestration combines content generation with a set of functions like templating and content filtering that are often required in business AI scenarios.
+
+Templating lets you compose a prompt with placeholders that are filled during the chat completion request.
+Content filtering lets you restrict the type of content that is passed to and received from a generative AI model.
+
+The Orchestration client allows you to send chat completion requests to generative AI models that are compatible with the orchestration service.
+More details about Orchestration can be found [here](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/orchestration-workflow).
```

Comment:
[req] try to avoid passive voice.
```suggestion
Find more details about the orchestration service [here](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/orchestration-workflow).
```